packages/curve-ui-kit/src/widgets/Disclaimer/Section.tsx

```diff
@@ -0,0 +1,55 @@
+import { ReactNode, Children } from 'react'
+
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+
+import { SizesAndSpaces } from 'curve-ui-kit/src/themes/design/1_sizes_spaces'
+
+const { Spacing } = SizesAndSpaces
+
+export const Header = ({ children }: { children?: ReactNode }) => (
+  <Stack
+    justifyContent="end"
+    sx={{
+      minHeight: '56px', // TODO: place as a constant somewhere.
```

Comment:
Can't we use `lineHeight` or `padding` here?

Reply:
Sadly not, as line height ads space before and after (top and bottom); in this case we only want space on top. However, I did replace this with `paddingBlockStart: Spacing.lg` which makes it responsive.